app/src/main/java/com/anytypeio/anytype/ui/settings/space/SpaceSettingsFragment.kt

```diff
@@ -402,4 +474,139 @@ fun SpaceSettingsScreen(
             Spacer(modifier = Modifier.height(16.dp))
         }
     }
+}
+
+@Composable
+@Preview
+fun SpaceSettingsScreenPreview() {
+    SpaceSettingsScreen(
+        spaceData = ViewState.Success(
+            data = SpaceSettingsViewModel.SpaceData(
+                spaceId = "ID",
+                createdDateInMillis = null,
+                createdBy = "1235",
+                network = "332311313131",
+                name = "Dream team",
+                icon = SpaceIconView.Placeholder,
+                isDeletable = true,
+                spaceType = PERSONAL_SPACE_TYPE,
+                permissions = SpaceMemberPermissions.OWNER
+            )
+        ),
+        onNameSet = {},
+        onDeleteSpaceClicked = {},
+        onFileStorageClick = {},
+        onPersonalizationClicked = {},
+        onSpaceIdClicked = {},
+        onNetworkIdClicked = {} ,
+        onCreatedByClicked = {},
+        onDebugClicked = {},
+        onRandomGradientClicked = {},
+        onManageSharedSpaceClicked = {},
+        onSharePrivateSpaceClicked = {}
+    )
+}
+
+@Composable
+fun PrivateSpaceSharing(
```

Comment:
I suppose it's better to move all Composable code to core-ui module

Reply:
yes, you're right, will do.

Reply:
done